src/lib/style/components/synapse_form_wrapper/_synapse-form-wrapper.scss

```diff
@@ -87,10 +87,10 @@
       padding-left: 0.8rem;
     }
     .panel {
-      background-color: $gray-light;
+      background-color: $SRC-gray-light;
     }
     .file-table {
-      @include calc(margin-bottom, #{$space-unit} * #{3});
+      @include calc(margin-bottom, "#{$space-unit} * #{3}");
```

Comment:
Couple of places where `sass` couldn't parse this while `node-sass` could. I believe that the change here yields the expected behavior.